form.yml.erb

```diff
@@ -15,6 +13,9 @@ form:
   - node_type
   - bc_vnc_resolution
   - bc_email_on_started
+cluster:
+    - "owens"
+    - "pitzer"
```

Comment:
Can you move this bit to the top. It's a little nit-picky, but it's just to keep uniformity with all the other apps and docs and so on. Lots of folks look at our apps for examples and best practices (such as they are).